flask-backend/app.py

```diff
@@ -19,10 +20,23 @@ def user_form():
         return render_template("user_form.html")
     # logic for post request
     else:
-        session["height"] = request.form["height"]
-        session["weight"] = request.form["weight"]
-        session["age"] = request.form["age"]
-        session["difficulty"] = request.form["difficulty"]
+        # collect user-entered data
+        user_data = request.form
```

Comment:
we could do something like this, only thing that would be challenging would be tryna get it to the front end, might look into js sessions